mqttclients/dotnet/MQTTnet.Client.Extensions/TelemetryMessage.cs

```diff
@@ -2,6 +2,7 @@
 
 public class TelemetryMessage<T>
 {
+    public ushort MessageId { get; set; }
```

Comment:
Why `ushort`? I get why unsigned, but curious the thought around the smaller number.

Reply:
It's defined by MQTTnet

https://github.com/dotnet/MQTTnet/blob/master/Source/MQTTnet/Packets/MqttPacketWithIdentifier.cs#L9